src/user-statuses/UserStatusScreen.js

```diff
@@ -157,7 +157,7 @@ export default function UserStatusScreen(props: Props): Node {
           <SelectableOptionRow
             key={item}
             itemKey={item}
-            title={item}
+            title={_(item)}
             selected={item === statusTextFromInputValue(textInputValue)}
             onRequestSelectionChange={itemKey => {
               setTextInputValue(_(itemKey));
```

Comment:
UserStatusScreen: Translate status-text suggestions
    
    Huh, I guess we hadn't been calling _, even though the strings
    appear in messages_en.json.

Yeah. I think in fact it's buggier than that -- the `setTextInputValue` call *does* translate. So you have an option that says "📅 In a meeting", even though you've chosen a UI language of say German; but then you select it and the text input gets filled with "📅 Bei einem Treffen".

Then further: when you choose that, you don't get the "selected" checkmark on the option as intended, because the `selected` logic is comparing to the untranslated text.

Those bugs are all fixed in the subsequent commit. But it'd be good to fix them all (including the latter one) in this separate commit. That can also simplify the subsequent commit that's meant to be about status emoji.

I'll try pushing a quick revision that does that.

Reply:
Ah, right! Thanks.